### PR TITLE
chore(e2e): headerbar + notifications + large display

### PR DESCRIPTION
**Pull request by jlame**
**What is the problem this PR is trying to solve?**
HeaderBar: get/click menuItem on label
List:  no Large display selector
Notification: no notification selector

**What is the chosen solution to this problem?**
HeaderBar: get/click menuItem by id
List: create Large display selector that use existing items/cells
Notification: create notification selector

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/e2e/component-objects/src/main/java/org/talend/component/HeaderBar.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+public class HeaderBar extends Component {
+
+    private static final Logger LOGGER = LogManager.getLogger(AppHeaderBar.class);
+
+    static final String HEADER_NAME = "AppHeaderBar";
+
+    static final String HEADER_SELECTOR = ".tc-header-bar";
+
+    static final String HELP_SELECTOR = "header-help";
```
Comment (tlnd-jbrun) [inline packages/components/e2e/component-objects/src/main/java/org/talend/component/HeaderBar.java]: It is not really a selector...Maybe it would be better to rename this

File: packages/components/e2e/component-objects/src/main/java/org/talend/component/HeaderBar.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    static final String USER_SELECTOR = "header-user";
```
Comment (tlnd-jbrun) [inline packages/components/e2e/component-objects/src/main/java/org/talend/component/HeaderBar.java]: the same : not really a selector

File: packages/components/e2e/component-objects/src/main/java/org/talend/component/HeaderBar.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    static final String USER_DROP_DOWN_SELECTOR = ".dropdown-menu.dropdown-menu-right";
+
+    static final String ABOUT_SELECTOR = "//li/a[@label='About']";
```
Comment (tlnd-jbrun) [inline packages/components/e2e/component-objects/src/main/java/org/talend/component/HeaderBar.java]: is it possible to avoid xpath request ?

File: packages/components/e2e/component-objects/src/main/java/org/talend/component/List.java [hunk 4]
```diff
@@ -56,20 +61,47 @@ public WebElement getAddButton() {
...
+     */
+    public WebElement getToolBarItemById(String id) {
+        return this.getElement().findElement(By.cssSelector(String.format(TOOL_BAR_ITEM_SELECTOR, id)));
+    }
+
     /**
      * Get the table display element manager.
      *
-     * @return Table the table element manager
+     * @return Large the table element manager
```
Comment (tlnd-jbrun) [inline packages/components/e2e/component-objects/src/main/java/org/talend/component/List.java]: why did you change the javadoc ?

File: packages/components/e2e/component-objects/src/main/java/org/talend/component/List.java [hunk 5]
```diff
@@ -56,20 +61,47 @@ public WebElement getAddButton() {
...
+    public Large getLarge() {
+        return new Large(driver);
+    }
+
     /**
      * Get the table display element manager identified by id.
      *
      * @param id The table id
-     * @return Table the table element manager
+     * @return Large the table element manager
```
Comment (tlnd-jbrun) [inline packages/components/e2e/component-objects/src/main/java/org/talend/component/List.java]: same thing

File: packages/components/e2e/component-objects/src/main/java/org/talend/component/Notification.java [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class Notification extends Component {
+
+    private static final String NOTIFICATION_NAME = "Notification";
+
+    private static final String NOTIFICATION_SELECTOR = ".tc-notification-container";
+
+    private static final String NOTIFICATION_MESSAGE = ".tc-notification-error .tc-notification-message";
```
Comment (tlnd-jbrun) [inline packages/components/e2e/component-objects/src/main/java/org/talend/component/Notification.java]: maybe it would be a good thingto differentiate these selectors ?

File: packages/components/e2e/component-objects/src/main/java/org/talend/component/list/large/Large.java [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+                .findElements(By.cssSelector(TABLE_ITEM_SELECTOR)) //
+                .stream() //
+                .map(webElement -> new ItemLarge(driver, webElement)) //
+                .collect(toList());
+    }
+
+    /**
+     * Scroll to top
+     */
+    public void scrollToTop() {
```
Comment (tlnd-jbrun) [inline packages/components/e2e/component-objects/src/main/java/org/talend/component/list/large/Large.java]: it might be a good thing to mutualize the different functions (use an argument for exemple)
scrollToTop
canScrollDown
getItem
hasItem

because the differences are really minimal with the classe Table

File: packages/components/e2e/component-objects/src/main/java/org/talend/component/HeaderBar.java (hunk 3 above)
Comment (tlnd-jbrun) [reply packages/components/e2e/component-objects/src/main/java/org/talend/component/HeaderBar.java]: li > a[label=About]

Comment (jlame) [reply packages/components/e2e/component-objects/src/main/java/org/talend/component/HeaderBar.java]: in fact it should be replace by a getBtnById something like that

File: packages/components/e2e/component-objects/src/test/java/org/talend/component/ListTest.java [hunk 8]
```diff
@@ -1,7 +1,14 @@
 package org.talend.component;
 
+import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
+import org.openqa.selenium.By;
```
Comment (jsomsanith) [inline packages/components/e2e/component-objects/src/test/java/org/talend/component/ListTest.java]: Clean unused imports

File: packages/components/e2e/component-objects/src/main/java/org/talend/component/List.java [hunk 9]
```diff
@@ -74,4 +108,19 @@ public Table getTable() {
...
+
+    /**
+     * Get the table display element manager.
+     *
+     * @return table the table element manager
+     */
+    public Display getDisplay() {
+        Large large = new Large(driver);
+        try {
+            large.getElementToScroll();
```
Comment (jsomsanith) [inline packages/components/e2e/component-objects/src/main/java/org/talend/component/List.java]: Why do you get the scroll element ?
Use `getElement()` instead please. Or use display selectors, I think there is a `tc-list-large` or classes like that

File: packages/components/e2e/component-objects/src/test/java/org/talend/component/ListTest.java [hunk 10]
```diff
@@ -29,4 +36,15 @@ public void should_get_table_with_id() {
         // then
         // should not throw NotFoundException
     }
+
+    @Test
+    public void should_get_table_using_display() {
+        // when
+        // should not throw NotFoundException
+        WebElement element = listObject.getDisplay().getElement();
```
Comment (jsomsanith) [inline packages/components/e2e/component-objects/src/test/java/org/talend/component/ListTest.java]: This var is not used

File: packages/components/e2e/component-objects/src/test/java/org/talend/component/display/LargeTest.java [hunk 11]
```diff
@@ -0,0 +1,110 @@
+package org.talend.component.display;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebElement;
+import org.talend.component.StorybookTest;
+import org.talend.component.list.Item;
+import org.talend.component.list.display.Cell;
```
Comment (jsomsanith) [inline packages/components/e2e/component-objects/src/test/java/org/talend/component/display/LargeTest.java]: Clean unused import

File: packages/components/e2e/component-objects/src/main/java/org/talend/component/AppHeaderBar.java [hunk 12]
```diff
@@ -152,4 +152,13 @@ public WebElement getHelp() {
     public void clickOnHelp() {
         this.getHelp().click();
     }
+
+    /**
+     * Retrieve an item by its id
+     * @param id
+     * @return the header bar element with the given id
+     */
+    public WebElement getItemById(String id) {
```
Comment (jsomsanith) [inline packages/components/e2e/component-objects/src/main/java/org/talend/component/AppHeaderBar.java]: Hmm not sure that this is the kind of thing we want. We create and maintain generic selectors so they follow any structure change of the component.
Ids should be used to get the component (ex: if you have 2 lists in the page, use the id to identify the list) but to work within the component, you should use the utility function we provide, or else you don't need the component-objects

Comment (jlame) [reply packages/components/e2e/component-objects/src/main/java/org/talend/component/AppHeaderBar.java]: well I did it because of the translation ... So i didn't want to depend on labels as they can change, didn't find better than that.

File: packages/components/e2e/component-objects/src/main/java/org/talend/component/Notification.java [hunk 13]
```diff
@@ -0,0 +1,43 @@
+package org.talend.component;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+import java.util.Optional;
```
Comment (jsomsanith) [inline packages/components/e2e/component-objects/src/main/java/org/talend/component/Notification.java]: Unused import
